lib/ansible/modules/network/cloudengine/ce_vxlan_arp.py

```diff
@@ -0,0 +1,695 @@
+            self.existing["host_collect_protocol"] = "bgp"
+        else:
+            self.existing["host_collect_protocol"] = None
+
+        if is_config_exist(self.config, "arp broadcast-suppress enable"):
+            self.existing["arp_suppress"] = "enable"
+        else:
+            self.existing["arp_suppress"] = "disable"
+
+    def get_end_state(self):
+        """get end state info"""
+
+        config = self.get_current_config()
+        evn_bgp_exist = is_config_exist(config, "evn bgp")
+        if evn_bgp_exist:
+            self.end_state["evn_bgp"] = "enable"
+        else:
+            self.end_state["evn_bgp"] = "disable"
+
+        if evn_bgp_exist:
+            if is_config_exist(config, "server enable"):
+                self.end_state["evn_server"] = "enable"
+            else:
+                self.end_state["evn_server"] = "disable"
+
+            self.end_state["evn_source_ip"] = get_evn_srouce(config)
+            self.end_state["evn_peer_ip"] = get_evn_peers(config)
+            self.end_state[
+                "evn_reflect_client"] = get_evn_reflect_client(config)
+
+        if is_config_exist(config, "arp collect host enable"):
+            self.end_state["host_collect_protocol"] = "enable"
+        else:
+            self.end_state["host_collect_protocol"] = "disable"
+
+        if is_config_exist(config, "host collect protocol bgp"):
+            self.end_state["host_collect_protocol"] = "bgp"
+        else:
+            self.end_state["host_collect_protocol"] = None
+
+        if is_config_exist(config, "arp broadcast-suppress enable"):
+            self.end_state["arp_suppress"] = "enable"
+        else:
+            self.end_state["arp_suppress"] = "disable"
+
+    def work(self):
+        """worker"""
+
+        self.check_params()
+        self.config = self.get_current_config()
+        self.get_existing()
+        self.get_proposed()
+
+        # deal present or absent
+        if self.evn_bgp or self.evn_server or self.evn_peer_ip or self.evn_source_ip:
+            self.config_evn_bgp()
+
+        if self.vbdif_name and self.arp_collect_host:
+            self.config_vbdif()
+
+        if self.host_collect_protocol:
+            self.config_host_collect_protocal()
+
+        if self.bridge_domain_id and self.arp_suppress:
+            self.config_bridge_domain()
+
+        if self.commands:
+            self.cli_load_config(self.commands)
+            self.changed = True
+
+        self.get_end_state()
+        self.results['changed'] = self.changed
+        self.results['proposed'] = self.proposed
+        self.results['existing'] = self.existing
+        self.results['end_state'] = self.end_state
+        if self.changed:
+            self.results['updates'] = self.updates_cmd
+        else:
+            self.results['updates'] = list()
+
+        self.module.exit_json(**self.results)
+
+
+def main():
+    """Module main"""
+
+    argument_spec = dict(
```

Comment:
For integers please:
use `type='int`
remove the `isdigit` check from `check_params`